week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -4,13 +4,19 @@
   Create a function which:
   - Accepts an array which contains all the students' names and their attendance counts
     (see tests to confirm how this data will be structured)
-  - Returns an array containing onl the names of the who have attended AT LEAST 8 classes
+  - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
-}
-
+function eligibleStudents(attendances) {
+     const filteredArr =[];
+     attendances.forEach(element=>{
```

Comment:
This works, well done :) To practice using the `filter` function, could you try also solving this using `filter`, rather than `forEach`?